components/CheckDocumentation/vacuum/InefficientIndexPhase.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+
+import { CheckDocs, CheckGuidanceProps, CheckTriggerProps } from "../../../util/checks";
+
+const InefficientIndexPhaseTrigger: React.FunctionComponent<CheckTriggerProps> = ({
+  config,
+}) => {
+  const threshold = config.settings["threshold_count"] as number;
+  const runPluralized = threshold === 1 ? 'run' : 'runs';
+  return (
+    <>
+      <p>
+        Detects when autovacuum runs in this server are forced to perform inefficient
+        multiple index scan phases due to limited configured memory, and creates an
+        issue with severity "warning". Triggers when at least <code>{threshold}</code>
+        autovacuum {runPluralized} in the last 24h have had multiple index scan phases.
+        Resolves automatically once fewer than <code>{threshold}</code> autovacuum {runPluralized}
```

Comment:
Uhmm any idea why this is happening? Or it's the old version screenshot?

![image](https://user-images.githubusercontent.com/911433/234440618-8a75ea50-fbf9-4b1c-bf5d-2ecc53c7b1fa.png)

Reply:
Ah, good catch. This needs a `{" "}` due to [whitespace handling in JSX](https://stackoverflow.com/a/32690647).